app/views/groups/_member.html.erb

```diff
@@ -0,0 +1,21 @@
+<li itemscope itemtype="http://schema.org/Code">
+  <div class="header">
+    <div class="header-content">
+      <%= link_to member.user, rel: 'member', itemprop: 'username' do %><%= member.user.username %><% end %>
+      <div class="remove">
+        <% if admin_member?(current_user, member.group) %>
+          <% if member.admin? %>
+            <% if admin_members(member.group).count > 1 %>
```

Comment:
This was a bit gnarly to follow. What do you think about condensing the "show the remove button" logic to a single `if`?

```erb
<% if admin_member?(current_user, member.group) %>
        
  <% if !member.admin? || (member.admin? && admin_members(member.group).count > 1) %>
    <%= link_to "Remove", group_member_path(member.id), method: :delete, data: { confirm: "Are you sure you want to remove this member?" }, class: 'button tiny alert radius' %>
  <% end %>

  <% if !member.admin? %>
    <%= link_to "Make Admin", make_admin_group_member_path(member.id), method: :post, class: 'button tiny radius' %>
  <% end %>

<% end %>
```

```ruby
!member.admin? || (member.admin? && admin_members(member.group).count > 1 )`
```
... could become ...

```ruby
!member.admin? || admin_members(member.group).count > 1
```

... but then we might be back in "what's going on here?" territory again.